jobs/blockhead/spec

```diff
@@ -15,3 +16,5 @@ properties:
     description: "Username for Blockhead broker"
   password:
     description: "Password for Blockhead broker"
+  port:
+    description: "Port for Blockhead broker to run on"
```

Comment:
can we set a default `3333` for the port in the spec and get rid of the port property in the manifest.

Reply:
I was going back and forth on this one. Blockhead already defaults the port. I am thinking we should make it required blockhead and default here, but I will make that change.